src/utils/metadata.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * This source file is part of the Swift.org open source project
+ *
+ * Copyright (c) 2021 Apple Inc. and the Swift project authors
+ * Licensed under Apache License v2.0 with Runtime Library Exception
+ *
+ * See https://swift.org/LICENSE.txt for license information
+ * See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+import { getSetting } from 'docc-render/utils/theme-settings';
+import { absoluteURL } from 'docc-render/utils/assets';
+
+const themeTitle = getSetting(['meta', 'title'], process.env.VUE_APP_TITLE);
+const DEFAULT_DESCRIPTION = 'There\'s never been a better time to develop for Apple platforms.';
```

Comment:
I know it's just a default, but we should double check this copy with the team before merging. The message might possibly sound too Apple-specific as-is.

Reply:
I deleted the default description and made sure description tags gets deleted if user navigates from a page with description to a page without it. I think it makes more sense like this since a page description should provide specific page value and if we don't have that information we shouldn't display a default value.

Reply:
2ff3d0dd1bca1ba340382c95e4e8fb83c75c6309